README.md

```diff
@@ -12,7 +12,7 @@ repositories {
 }
 
 dependencies {
-    implementation("io.github.nomisrev:saga:0.1.3")
+    implementation("io.github.nomisrev:Saga-jvm:1.0.1")
```

Comment:
This should also not be necessary :/ So something is broken in my release process.

Reply:
I picked up the dependency from here
https://mvnrepository.com/artifact/io.github.nomisrev/Saga-jvm
but now you mention that, I also see it here
https://mvnrepository.com/artifact/io.github.nomisrev/Saga
and here
https://mvnrepository.com/artifact/io.github.nomisrev/saga

Reply:
Sorry for my slow replies @rinkledink. Could you check if `implementation("io.github.nomisrev:Saga:1.0.1")` resolves correctly for you?

Reply:
yeah, it resolves